### ISSUE TITLE
Availability of pretrained weights

### ISSUE THREAD

**Issue by Junjie-Zhu** [Vincentx15/atomsurf#6]
Hi there,

Excellent work! 

I am interested in applying your model to some of my own data, but I noticed that the pretrained checkpoints are currently missing from the repository. Would it be possible to provide a download link for the pretrained weights?

**Comment by Vincentx15** [comment]
Hello,

Thanks for your interest in our work ! In the paper, we propose our encoder and train it on a set of specific tasks. We do not pre-train our model, so we don't really ship pre-trained weights.

If you're interested, we can share the weights of the model trained to predict protein interactions on the PINDER dataset.

Moreover, @eaglev-sci actually played a bit with pre-training so we have a model pre-trained on a masking objective, but we do not have extensive validation for this model yet. 

Let me know what you would rather have !

**Comment by Junjie-Zhu** [comment]
Hi,

Thanks for the follow-up.

To clarify, I am exactly interested in the model weights trained on the specific tasks. 

I am currently working on protein interaction prediction, so access to the weights trained on the PINDER dataset would be incredibly helpful for my research!

**Comment by VictorGtn** [comment]
Hi,

I'm Vincent's PhD student, I worked on [AlphaSurf](https://openreview.net/forum?id=rBjKl58v54), an extension to AtomSurf that bypasses the need for precomputed fine-grained MSMS meshes. The code is available at [github.com/VictorGtn/alphasurf](https://github.com/VictorGtn/alphasurf).

We now provide the PINDER-Pair-trained checkpoint in the repo at `alphasurf/tasks/pinder_pair/ckpt/last.ckpt`. You can use the `embed.py` script to generate per-residue graph embeddings and per-vertex surface embeddings from any PDB file:

```bash
python tasks/inference/embed.py --ckpt ../pinder_pair/ckpt/last.ckpt --pdb protein.pdb
```

Run with `--help` to see all options (batching, `--pdb-dir`, `--output-dir`, etc.).

This generates alpha complex surfaces on the fly, so no precomputation is needed. See the **Installation** and **Inference** section of the [README](https://github.com/VictorGtn/AlphaSurf/blob/main/README.md) for full details including setup instructions (symlink, CGAL bindings, etc.). 

Hope this helps! Feel free to open a new issue on AlphaSurf's repo if you run into any problems.

**Comment by Vincentx15** [comment]
Hello, 

Thanks again for your issue. Did the comment above address it ?

**Comment by Junjie-Zhu** [comment]
Thank!

My questions are addressed.
